lib/livebook/hubs/team_client.ex

```diff
@@ -266,18 +265,42 @@ defmodule Livebook.Hubs.TeamClient do
     }
   end
 
+  defp put_agent_key(deployment_group, agent_key) do
+    deployment_group = remove_agent_key(deployment_group, agent_key)
+    agent_keys = [agent_key | deployment_group.agent_keys]
+
+    %{deployment_group | agent_keys: Enum.sort_by(agent_keys, & &1.id)}
+  end
+
+  defp remove_agent_key(deployment_group, agent_key) do
+    %{
+      deployment_group
+      | agent_keys: Enum.reject(deployment_group.agent_keys, &(&1.id == agent_key.id))
+    }
+  end
+
+  defp build_agent_key(agent_key) do
+    %Teams.AgentKey{
+      id: to_string(agent_key.id),
+      key: agent_key.key,
+      deployment_group_id: to_string(agent_key.deployment_group_id)
```

Comment:
All of those ids should be strings already, no?

Reply:
Yeah, good call, I forgot to remove these here